gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,5 @@
-#Fri Aug 04 11:59:26 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionSha256Sum=fa4873ae2c7f5e8c02ec6948ba95848cedced6134772a0169718eadcb39e0a2f
```

Comment:
Why a checksum has been removed?

Reply:
This is just the result of running ./gradlew wrapper.  So the wrapper task removed this.

Reply:
Yes, the wrapper don't generate it automatically. However, as it was here before, someone put it there explicitly and it would be good to keep it doing just the Gradle wrapper version  :).